spec/index.bs

```diff
@@ -9339,6 +9339,14 @@ The determination is based on |descriptor|.{{GPURenderPipelineDescriptor/multisa
 
 Issue: fill out this section
 
+### Output Merging ### {#output-merging}
```

Comment:
Tentatively added this new section, is it in the right spot? And what should it be called?

Reply:
I'm landing this but this bit will conflict with #2162. We can resolve this question there since it's the editorial one for this section.